test/workflows/components/workflows.libsonnet

```diff
@@ -396,6 +400,12 @@
               "--context=dir://" + srcDir + "/python",
               "--destination=" + "527798164940.dkr.ecr.us-west-2.amazonaws.com/kfserving/sklearnserver:$(PULL_BASE_SHA)",
             ]),  // build-sklearnserver
+            $.parts(namespace, name, overrides).e2e(prow_env, bucket).buildTemplate("build-pmmlserver", kanikoExecutorImage, [
+                "/kaniko/executor",
+                "--dockerfile=" + srcDir + "/python/pmml.Dockerfile",
+                "--context=dir://" + srcDir + "/python",
+                "--destination=" + "527798164940.dkr.ecr.us-west-2.amazonaws.com/kfserving/pmmlserver:$(PULL_BASE_SHA)",
```

Comment:
@PatrickXYS Can you help add this new image to ECR? thanks.

Reply:
Added